From: Igor Gnatenko <ignatenkobrain@fedoraproject.org>
Date: Wed, 29 Mar 2017 05:20:51 +0000 (+0200)
Subject: qt: pass fmt to gpgrt_asprintf()
X-Git-Url: http://git.gnupg.org/cgi-bin/gitweb.cgi?p=gpgme.git;a=commitdiff_plain;h=5d4f977dac542340c877fdd4b1304fa8f6e058e6

qt: pass fmt to gpgrt_asprintf()

* lang/qt/src/dn.cpp (parse_dn_part): Add fmt argument instead of
using name directly.

GnuPG-Bug-Id: 3023
Signed-off-by: Igor Gnatenko <ignatenkobrain@fedoraproject.org>
Signed-off-by: Andre Heinecke <aheinecke@intevation.de>
---

diff --git a/lang/qt/src/dn.cpp b/lang/qt/src/dn.cpp
index f9fb2f6..4310ad6 100644
--- lang/qt/src/dn.cpp
+++ lang/qt/src/dn.cpp
@@ -167,7 +167,7 @@ parse_dn_part(DnPair *array, const unsigned char *string)
     for (unsigned int i = 0; i < numOidMaps; ++i)
         if (!strcasecmp((char *)p, oidmap[i].oid)) {
             free(p);
-            gpgrt_asprintf(&p, oidmap[i].name);
+            gpgrt_asprintf(&p, "%s", oidmap[i].name);
             break;
         }
     array->key = p;
